test/integration/vue.spec.ts

```diff
@@ -12,6 +12,23 @@ interface Error {
   rawMessage: string;
 }
 
+const templateCompilers = [
+  'vue-template-compiler',
+  'nativescript-vue-template-compiler'
+] as const;
+
+interface VueEnabledOption {
+  enabled: true;
+  compiler: typeof templateCompilers[number];
+}
+
+const vueEnabledOptions = templateCompilers.reduce(
+  (list, templateCompiler) => {
+    return list.concat({ enabled: true, compiler: templateCompiler });
+  },
+  [] as VueEnabledOption[]
+);
+
```

Comment:
This isn't very readable, please use something along the lines of
```typescript
const vueEnabledOptions = templateCompilers.map(compiler => ({ enabled: true, compiler}))
```

Reply:
Code block was removed